docs/user-guide/deployments.md

```diff
@@ -6,7 +6,7 @@
 
 ## What is a _Deployment_?
 
-A _Deployment_ provides declarative updates for Pods and ReplicationControllers.
+A _Deployment_ provides declarative updates for Pods and ReplicaSets.
 Users describe the desired state in a Deployment object, and the deployment
```

Comment:
I would address the doc to the user rather than refer to them in the third person -- this doc is for them